dashboard/modules/reporter/reporter_agent.py

```diff
@@ -592,9 +603,12 @@ def _get_workers(self):
         if raylet_proc is None:
             return []
         else:
-            workers = {
-                self._generate_worker_key(proc): proc for proc in raylet_proc.children()
-            }
+            workers = {}
```

Comment:
Should we do this for windows only?

Reply:
I don't think so. While [PEP 397](https://peps.python.org/pep-0397/) does mention windows, there is a [launcher for other operating systems](https://snarky.ca/introducing-the-python-launcher-for-unix/). The problem found here is subtle and by making it windows-only may trip up users on other platforms.